frontend/src/features/mapSideBar/containers/MapSideBarContainer.tsx

```diff
@@ -395,6 +428,23 @@ const MapSideBarContainer: React.FunctionComponent<IMapSideBarContainerProps> =
         ) : (
           <Spinner animation="border"></Spinner>
         );
+      case SidebarContextType.ADD_SUBDIVISION_LAND:
+        return (
+          <LandForm
+            setMovingPinNameSpace={setMovingPinNameSpace}
+            formikRef={formikRef}
+            handleGeocoderChanges={handleGeocoderChanges}
+            handlePidChange={handlePidChange}
+            handlePinChange={handlePinChange}
+            findMatchingPid={findMatchingPid}
+            isPropertyAdmin={keycloak.hasClaim(Claims.ADMIN_PROPERTIES)}
+            setLandComplete={setShowCompleteModal}
+            setLandUpdateComplete={setShowUpdatedModal}
+            initialValues={
+              parcelDetail ?? { ...getInitialValues(), propertyTypeId: PropertyTypes.SUBDIVISION }
```

Comment:
needed to seperate this out from the other AddLand... states since I need to pass in the propertytype of the parcel. (default is 0).